great_expectations/expectations/metrics/column_aggregate_metrics/column_quantile_values.py

```diff
@@ -142,6 +144,15 @@ def _sqlalchemy(
                 sqlalchemy_engine=sqlalchemy_engine,
                 table_row_count=table_row_count,
             )
+        elif dialect.name.lower() == "awsathena":
+            return _get_column_quantiles_athena(
+                column=column,
+                quantiles=quantiles,
+                allow_relative_error=allow_relative_error,
```

Comment:
The `allow_relative_error` kwarg doesn't appear to be used in your `_get_column_quantiles_athena` func

Reply:
done